app/celery/v3/notification_tasks.py

```diff
@@ -30,6 +30,39 @@
 logger = get_task_logger(__name__)
 
 
+def get_default_sms_sender_id(service_id: str) -> str:
+    """
+    Retrieve the default SMS sender ID for a given service.
+
+    Parameters:
+    - service_id (str): The UUID for the service whose default SMS sender ID is being retrieved.
+
+    Returns:
+    - str: The ID of the default SMS sender.
+
+    Raises:
+    - RuntimeError:
+        If no default SMS sender ID is found.
+        If there is an unexpected error during the database query.
+        If the SMS sender ID is None after retrieving a sender that was marked as default.
```

Comment:
This needs to be updated since you've not returning RuntimeError anymore.

Reply:
done